test/testSetup.ts

```diff
@@ -60,7 +60,7 @@ export class TestSetup {
       // await utilities.executeCommand(workbench, `sfdx force:org:delete -u ${this.scratchOrgAliasName} --noprompt`);
 
       // The Terminal view can be a bit unreliable, so directly call exec() instead:
-      await exec(`sfdx force:org:delete -u ${this.scratchOrgAliasName} --noprompt`);
+      await exec(`sfdx org:delete:scratch --target-org ${this.scratchOrgAliasName} --no-prompt`);
```

Comment:
If we don't want to use terminal view, how about cleaning up the commented code above?

Reply:
And why are we holding on to code that no longer works?
Is there a work item to look into why so it can be re-enabled at some point?

Reply:
Typically for E2E tests, if one method doesn't work we just find a workaround.  Since using `exec()` is a good workaround, we are likely not switching back to using Terminal view (as testing the CLI is not the purpose of the E2E tests so it doesn't matter how we run the commands).  I can clean up the commented code.

Reply:
I see that one block of commented code has the functionality of deleting the folder where the project was located during the tear down process.  We don't need that anymore because each E2E test is now run individually on a separate VM, so it doesn't matter if we do that cleanup.